src/corona_hakab_model/common/util.py

```diff
@@ -25,7 +30,11 @@ def off_binom(a, c, b):
     if len(args) == 1:
         return const_dist(*args)
     if len(args) == 2:
-        return uniform_dist(*args)
+        if type(args[0]) == list and type(args[1]) == list:
```

Comment:
do not use "type(...) =="
If you want to do this check use isinstance

Reply:
fixed